worlds/__init__.py

```diff
@@ -87,6 +91,7 @@ def load(self) -> bool:
             file_like.seek(0)
             import logging
             logging.exception(file_like.read())
+            failed_world_loads.append(os.path.basename(self.path).rsplit(".", 1)[0])
```

Comment:
I don't really like this because world names may not fuzzy match well against their file path e.g. HK, kh2.

But I don't know a better alternative so the code looks good enough to me 🤷

Reply:
Maybe we can change this once we add metadata to the worlds, but that's not a thing yet.